projects/lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-remotedata",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Actually, this would be `2.1.0` because is a feature.
Also, change the version in `projects/lib/package.json` please.

Reply:
ok